modules/microshift-low-latency-tuned-profile.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-tuned-profile_{context}"]
+= Configuring the {microshift-short} TuneD profile
+
+Configure a TuneD profile for your host to use low latency with {microshift-short} workloads using the `microshift-baseline-variables.conf` configuration file provided in the {op-system-base-full} `/etc/tuned/` host directory after you install the `microshift-low-latency` RPM package.
+
+.Prerequisites
+
+* You have root access to the cluster.
+* You installed the `microshift-low-latency` RPM package.
+* Your {op-system} host has TuneD installed. See link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/monitoring_and_managing_system_status_and_performance/getting-started-with-tuned_monitoring-and-managing-system-status-and-performance#the-location-of-tuned-profiles_getting-started-with-tuned[Getting started with TuneD] (RHEL documentation).
+
+.Procedure
+
+. You can use the default `microshift-baseline-variables.conf` TuneD profile in the `/etc/tuned/` directory profile, or create your own to add more tunings.
++
+.Example `microshift-baseline-variables.conf` TuneD profile
+[source,text]
+----
+# Isolate cores 2-7 for running application workloads
+isolated_cores=2-7 # <1>
+
+# Size of the hugepages
+hugepages_size=2M # <2>
+
+# Number of hugepages
+hugepages=0
+
+# Additional kernel arguments
+additional_args= # <3>
+
+# CPU set to be offlined
+offline_cpu_set= # <4>
+----
+--
```

Comment:
I think you need a `+` here. The callouts look out of line and the next step restarts at 1 in the preview. 
```suggestion
+
--
```